README.md

```diff
@@ -346,6 +351,35 @@ command:
 xcodebuild build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -project "Mozilla VPN.xcodeproj"
 ```
 
+#### Building with CMake (Experimental)
+
+We also support building from sources for iOS using CMake. This is a work in progress and these
+instructions may be subject to change as we figure out what works and what doesn't.
+
+1. On iOS, we compile the app using
+[Xcode](https://developer.apple.com/xcode/) version 12 or higher and [Qt](https://www.qt.io/download)
+version 6.3.2.
+
+2. We use `qt-cmake` from the Qt installation to configure the Xcode project.
+```bash
+/Users/example/Qt/6.3.2/ios/bin/qt-cmake . -B build -GXcode
+```
+
+Some variables that might be useful when configuring the project:
+ - `BUILD_ADJUST_SDK_TOKEN=<SDK Token>`: can be set to enable the use of the Adjust telemetry
+   and attribution data collection.
+ - `BUILD_IOS_APP_IDENTIFIER=<App Identifier>`: can be set to change the application bundle
+   identifier. This defaults to `org.mozilla.ios.FirefoxVPN` if not set.
+ - `BUILD_VPN_DEVELOPMENT_TEAM=<Development Team ID>`: can be set to change the development
+   team used for Xcode certificates. This defaults to `43AQ936H96` if not set.
+ - `CMAKE_OSX_SYSROOT="iphonesimulator"` to build for the iOS simulator.
+
+3. Open the generated Xcode project with `open build/Mozilla\ VPN.xcodeproj`
+
+4. Select the `mozillavpn` target and `Any iOS Device (arm64)` as the build configuration.
```

Comment:
Do we want to recommend the `Any iOS Device` setting? For building for testing, don't we want the specific device/simulator?

Reply:
I think the `Any iOS Device` setting is how Xcode tries to build for real devices, and we are trying to make sure that this setting matches what cmake is expecting at configuration time via the `CMAKE_OSX_SYSROOT` option. Unwinding the hardcoded nature of this sysroot setting is going to require some work on supporting multiple-architecture builds.